src/View/Notification.elm

```diff
@@ -106,16 +107,25 @@ notificationStatusView ( context, currentUser, status, { type_, accounts } ) =
 notificationFollowView : CurrentUser -> NotificationAggregate -> Html Msg
 notificationFollowView currentUser { accounts } =
     let
-        profileView account =
-            div [ class "status follow-profile" ]
-                [ Common.accountAvatarLink False account
-                , div [ class "username" ] [ Common.accountLink False account ]
-                , formatContent account.note []
-                    |> div
-                        [ class "status-text"
-                        , onClick <| Navigate ("#account/" ++ (toString account.id))
+        profileView : AccountNotificationDate -> Html Msg
+        profileView accountAndDate =
+            let
+                account =
+                    accountAndDate.account
```

Comment:
Nit: you can just destructure `account` and `created_at` in the arg: `profileView { account, created_at } =`.